src/common.h

```diff
@@ -41,7 +41,6 @@ inline const char *mode_strings[] = {
 };
 
 struct AppContext {
-  size_t frame          = 0;
```

Comment:
Why not just keep frame instead? or rename to spp maybe, since that is actually more descriptive?